.commitlintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["@commitlint/config-angular"]
```

Comment:
I'm unfamiliar with the angular commit conventions. Though it looks somewhat similar to the SFRA team commit conventions.

Reply:
I think the commitlint's angular flavor is the most used changelog flavor out there, so it's probably best to go with it. If I remember correctly, the whole commit linting stuff started with the angular project back in the good old days.